__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.0"
```

Comment:
The version isn't being read from here. 

https://github.com/openedx/codejail-includes/pull/1/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R11